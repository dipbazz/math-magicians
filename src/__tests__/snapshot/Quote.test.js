import React from 'react';
import renderer from 'react-test-renderer';
import Quote from '../../components/Quote';

it('should render quote correctly', () => {
  const tree = renderer
    .create(<Quote />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
