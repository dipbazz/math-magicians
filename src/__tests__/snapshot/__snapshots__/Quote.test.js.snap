// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render quote correctly 1`] = `
<p>
  Mathematics is not about numbers, equations, computations, or algorithms: it is about understanding. â€“William Paul Thurston
</p>
`;
